packages/client/src/generation/TSClient/common.ts

```diff
@@ -328,8 +375,10 @@ type IsObject<T extends any> = T extends Array<any>
 ? False
 : T extends Date
 ? False
-: T extends Buffer
-? False
+${
+  // Edge runtime doesn't have Buffer type
+  edge ? '' : `: T extends Buffer\n? False`
```

Comment:
Nice catch, thanks.

Reply:
This has been changed in the latest release, we now use `Uint8Array` so we can be runtime agnostic, this can now be removed.